bin/webpack.js

```diff
@@ -1,21 +1,71 @@
 #!/usr/bin/env node
+function runCommand(command, options) {
+	const cp = require("child_process");
+	return new Promise((resolve, reject) => {
+		const executedCommand = cp.spawn(command, options, {
+			stdio: "inherit"
+		});
+
+		executedCommand.on("error", error => {
+			reject(error);
+		});
+
+		executedCommand.on("exit", code => {
+			if (code === 0) {
+				resolve(true);
+			} else {
+				reject();
+			}
+		});
+	});
+}
 
 let webpackCliInstalled = false;
 try {
 	require.resolve("webpack-cli");
 	webpackCliInstalled = true;
-} catch (e) {
+} catch (err) {
 	webpackCliInstalled = false;
 }
 
-if (webpackCliInstalled) {
-	require("webpack-cli"); // eslint-disable-line node/no-missing-require, node/no-extraneous-require, node/no-unpublished-require
+if (!webpackCliInstalled) {
+	const path = require("path");
+	const fs = require("fs");
+	const inquirer = require("inquirer");
+	const isYarn = fs.existsSync(path.resolve(process.cwd(), "yarn.lock"));
+
+	let packageManager;
+	let options = [];
```

Comment:
We can initialize options with ["install", "-D", "webpack-cli"] and we can then change `install` to `add` if yarn is installed. I think that will increase v8 optimization(we are not assigning options array multiple times).. Note that both yarn and npm support -D flag.

Reply:
Alright, i was always used to --save-dev that i forgot -D existed. I'm going to do that.